modules/apps/asset/asset-list-web/src/main/java/com/liferay/asset/list/web/internal/display/context/EditAssetListDisplayContext.java

```diff
@@ -610,6 +610,8 @@ public Map<String, Object> getData() {
 				return liferayPortletResponse.getNamespace() +
 					"openSelectSegmentsEntryDialog";
 			}
+		).put(
```

Comment:
I think we are not using this value in the JS components, so I think is better not to pass the value through the `getData` method. Maybe it's easier and safer to declare a public `getSegmentsConfigurationURL` and use it directly in the JSP.

Maybe we can do the same for `_isSegmentationEnabled` ? 🤔 
Just declare it public and use it both in the `getData` and in the JSP?
 cc @YurenaC